eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -173,6 +175,20 @@ jobs:
         customPrepArgs="${customPrepArgs} --no-sdk --no-artifacts"
         prepSdk=false
       fi
+
+      if [[ '${{ parameters.runOnline }}' == 'False' ]]; then
+        smokeTestFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json"
```

Comment:
Isn't this already in the NuGet.config being used?

Reply:
No, the pre-defined NuGet.config is using `https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json`, which is the "mirrored" location of packages from nuget.org.